### ISSUE TITLE
[BUG] Cruise Control Failure

### ISSUE THREAD

**Issue by nile-roth** [rlchavez3/cs386-a2.1#2]
Vehicle Motor is not adjusted when in "self-driving" mode. The car is still able to steer to avoid obstacles, but does not accelerate

**Comment by Jacob-Olaffson** [comment]
Yes, this seems important. We should implement a fix as soon as possible.

**Comment by rlchavez3** [comment]
@Jacob-Olaffson what should we do to fix this?
